lib/utilities.js

```diff
@@ -131,5 +180,6 @@ function isIE(version) {
 module.exports = {
     formatAttributes: formatAttributes,
     formatDOM: formatDOM,
-    isIE: isIE
+    isIE: isIE,
```

Comment:
Omit trailing comma as that is not supported in browsers like IE11

Also, once `CASE_SENSITIVE_TAGS` is moved to `lib/constants.js`, we should no longer export it here